bot/extensions/wikipedia_cog.py

```diff
@@ -0,0 +1,86 @@
+from discord.ext.commands import Cog, hybrid_command
+import discord
+from urllib.request import urlopen
+from urllib.parse import quote_plus
+from json import loads
+
+
+def search_results(search):
+        url_encode = quote_plus(search)
+        url = "https://en.wikipedia.org/w/api.php?action=opensearch" \
+              "&format=json&limit=3&namespace=0&search=" + url_encode
+        with urlopen(url) as url:
+            return loads(url.read())
+
+
+class Buttons(discord.ui.View):
+    def __init__(self, search, result):
+        super().__init__()
+        self.search = search
+        self.result = result
+        self.value = None
```

Comment:
I don't understand the point of this attribute. You assigned True to it but never use it anywhere. Maybe I don't see the purpose, if so can you explain it? If it's indeed unused, remove it.